app/components/GenericPlatform.tsx

```diff
@@ -127,39 +127,45 @@ export const GenericPlatform = ({ platFormGroupSpec, platform }: PlatformProps):
       const state = `${platform.path}-` + generateUID(10);
       const providerPayload = (await platform.getProviderPayload({ state, window, screen, waitForRedirect })) as {};
 
-      const verified: VerifiableCredentialRecord = await fetchVerifiableCredential(
-        iamUrl,
-        {
-          type: platform.platformId,
-          types: selectedProviders,
-          version: "0.0.0",
-          address: address || "",
-          proofs: providerPayload,
-          rpcUrl,
-        },
-        signer as { signMessage: (message: string) => Promise<string> }
-      );
-      // because we provided a types array in the params we expect to receive a
-      // credentials array in the response...
-      const vcs =
-        verified.credentials
-          ?.map((cred: CredentialResponseBody): Stamp | undefined => {
-            if (!cred.error) {
+      // This array will contain all providers that new validated VCs
+      let vcs: Stamp[] = [];
```

Comment:
Nit: might be a smidge cleaner to keep `.map/.filter` https://github.com/gitcoinco/passport/pull/728/files#diff-76a78c07ce4bb2a6bbd9387f6e57db7df025cb43927cb966f5b1eca732c9d097L155 methods as opposed to using let here

Reply:
I totally agree! The vcs weren't properly filtering with map and filter though, so we went with the for loop.